src/otx/v2/adapters/torch/mmengine/engine.py

```diff
@@ -627,3 +626,15 @@ def export(
         )
 
         return exporter.export()
+
+    def _update_codebase_config(self, codebase: str | None, task: str | None, deploy_config_dict: dict) -> None:
```

Comment:
yes, this is related to mmX deployment (export-related) function. I think we need a better mechanism loading YAML instead of patching things in `_update_config` as it's quite messy and error-prone patching things in functions.